edit-post/assets/stylesheets/main.scss

```diff
@@ -44,6 +44,11 @@
 	}
 }
 
+body.dragging * {
```

Comment:
Is the `*` necessary?

Reply:
Good point. I believe so but will double check to make sure.

Reply:
Nope, not necessary. Good call. Removed.